__tests__/e2e/bin/setup-env.sh

```diff
@@ -17,7 +17,7 @@ done
 vip dev-env destroy --slug=e2e-test-site
 
 # Create and run test site
-vip --slug=e2e-test-site dev-env create --title="E2E Testing site" --phpmyadmin --mu-plugins="${pluginPath}" --wordpress="5.9" --multisite=false --client-code="${clientCodePath}"
+vip --slug=e2e-test-site dev-env create --title="E2E Testing site" --mu-plugins="${pluginPath}" --wordpress="5.9" --multisite=false --client-code="${clientCodePath}"
```

Comment:
should WP be bumped here to the latest?

Reply:
We do that later:

https://github.com/Automattic/vip-go-mu-plugins/blob/develop/__tests__/e2e/bin/setup-env.sh#L34